docs/guides/setup/configuring_metadata_stores/components/_install_boto3_with_pip.mdx

```diff
@@ -0,0 +1,15 @@
+Python interacts with AWS through the `boto3` library.  Great Expectations makes use of this library in the background when working with AWS.  Therefore, although you will not need to use `boto3` directly, you will need to have it installed into your virtual environment.
```

Comment:
Rephrase idea? Just remove the sentence about python interacting with aws. Just start with "Great Expectations makes use of `boto3` library when working with AWS [... ]"